### ISSUE TITLE
As a user I want to be able to use numbers in "Set data type other" input field

### ISSUE THREAD

**Issue by rutgervandeelen** [Donders-Institute/data-streamer#72]
but not starting with number.

**Comment by marcelzwiers** [comment]
And also dashes ('-') and underscores ('_') I would say (e.g. 'audio-left' or 'audio_left'), possibly even also '+' and '&' (to allow for e.g. 'audio+video' or 'audio&video').

**Comment by rutgervandeelen** [comment]
The `data type other` input field explanation has become:
```
Should be lower case string, optionally with numbers, dashes ('-'), underscores ('_'), and '&amp;'. Example: 'eyetracker' or 'audio-left'
```
The regular expression for this input is:
```
^[a-z][-a-z0-9_\&]+$
```
In other words, start with lowercase characters. Next, allow as many lowercase characters, numbers dashes, underscores, and ampersands as needed.

**Comment by marcelzwiers** [comment]
Ampersands in the filename? Is that not undesirable for scripting, i.s. doesn't that need escaping / quoting (at least in windows)?

**Comment by rutgervandeelen** [comment]
You are right. Although it was originally your idea to allow ampersands (e.g. `audio&video`, see above), it is better to not introduce them. I have disabled the use of '&' in `data type other`.

**Comment by marcelzwiers** [comment]
Yes, my own suggestion, but it was meant as a discussion point. But in hindsight I don't think it was a great idea.

**Comment by rutgervandeelen** [comment]
Fixed in release 3.0.0
